source/hardware/controllers/numark_mixtrack_platinum_fx.rst

```diff
@@ -0,0 +1,363 @@
+   "5", "Cue Gain", "Adjusts the volume for headphone cueing."
+   "6", "Master Output LEDs", "Displays the audio level going to the Master Output."
+   "7", "Load", "Press one of these buttons while a track is selected to assign it to Deck 1 or 2, respectively."
+   "", ":hwlabel:`2X Press` Track Cloning", "Quickly double press a Load button to clone the track of the adjacent deck."
+   "", ":hwlabel:`SHIFT` Eject", "Removes the track from related deck."
+   "8", "Level", "Adjusts the pre-fader, pre-EQ audio level of the corresponding channel."
+   "9", "High EQ", "Controls the treble frequencies for the individual channels "
+   "10", "Mid EQ", "Controls the mid-range frequencies for the individual channels."
+   "11", "Low EQ", "Controls the bass frequencies for the individual channels."
+   "12", "Filter", "Adjusts the amount of the filter effect. Turning the knob left and right will produce a lowpass filter and high-pass filter, respectively."
+   "13", "Cue/PFL", "Sends pre-fader audio to the cue channel forheadphone monitoring. "
+   "", ":hwlabel:`SHIFT` Slip Mode", "Toggles slip mode on/off for related deck."
+   "14", "Channel Fader", "Adjusts the volume of the individual channels in the software."
+   "15", "Crossfader", "Controls the blend between the two decks"
+   "16", "Pitch Fader", "This controls the speed of the music. Moving towards the + will speed the music up, while moving towards the – will slow it down."
+   "17", "Pitch Bend Down", "Press and hold to momentarily reduce the speed of the track."
+   "", ":hwlabel:`SHIFT` Pitch Range", "Hold Shift and this djust Pitch Range"
+   "18", "Pitch Bend Up", "Press and hold to momentarily reduce the speed of the track."
+   "", ":hwlabel:`SHIFT` Keylock", "Hold Shift and this button to adjust Keylock."
+   "19", "Beats Multiplier", "Increases and decreases the rate of the effects to the beat. When using a time-based effect, this knob adjusts its time division."
+   "", ":hwlabel:`SHIFT` FX Macros", "Selects and controls FX Macros parameters."
+   "20", "FX Wet/Dry Knob", "Turn this knob to adjust the wet/dry mix of the effects."
+   "21", "Software FX", "Press one of these buttons to select a software effect.
+
+   • HPF (High pass filter)
+   • LPF (Low pass filter)
+   • Flanger
+   • Echo
+   • Reverb
+   • Phaser"
+   "21", ":hwlabel:`SHIFT` Multiple FX (at once)", "Hold shift and select/unselect multiple FX buttons to be active at once."
+   "22", "FX Toggle On / Off:", "Push up on the toggle switch to latch (lock) the FX in the on position. Push down on the toggle switch to turn the FX on momentarily. When the toggle switch is in the middle position, the FX will be off."
+   "23", "Tap BPM", "Press this a minimum of 4 times to manually enter a new BPM. The software will ignore the track's BPM and follow your manually entered tempo.
+
+   *See Tapping Logic*"
```

Comment:
hint: IIRC you can link to headers in this document:
* add a ref to the header
* link to it with
   ```
   :ref:`LinkTitle<ref-target>`
   ```

For example see `reloop-terminal-mix-series-options` in https://github.com/mixxxdj/manual/blob/2b3021375e075b3a1df77a4a8aa17705a2fefd77/source/hardware/controllers/reloop_terminal_mix_series.rst?plain=1

Reply:
Done on all the lines with see modes.